data/mods/MindOverMatter/powers/vitakinesis.json

```
@@ -474,5 +475,34 @@
     "effect_str": "EOC_VITA_SUPER_HEAL_DRAIN",
     "shape": "blast",
     "flags": [ "SILENT" ]
+  },
+  {
+    "id": "vita_limb_restore",
+    "type": "SPELL",
+    "name": "[Ψ]Ameliorate Extremity",
+    "description": "You sink your mind the deepest within yourself you can possibly get, and enter a meditative trance to rejuvenate lost limbs, one at a time.  Be sure to remove any prosthetics before doing this, it's hard to work around them.",
+    "message": "You sink your mind deep within your body and enter a trance.",
+    "teachable": false,
+    "valid_targets": [ "self" ],
+    "spell_class": "VITAKINETIC",
+    "skill": "metaphysics",
+    "flags": [ "CONCENTRATE", "SILENT", "NO_HANDS", "NO_LEGS", "NO_EXPLOSION_SFX" ],
+    "effect": "effect_on_condition",
+    "effect_str": "EOC_VITAKIN_RESTORE_LIMB_SELECTOR",
+    "extra_effects": [ { "id": "psionic_drained_difficulty_ten", "hit_self": true } ],
+    "shape": "blast",
+    "difficulty": 9,
```

```suggestion
    "difficulty": 10,
```
Based on the practice difficulty, I assume this is meant to be a Difficulty 10 power.